### ISSUE TITLE
Error after updating to Collabora 3.3.0 in nextcloud

### ISSUE THREAD

**Issue by ralcazar-oeg** [nextcloud/richdocuments#487]
**Describe the bug**
After updating to 3.3.0 (in apps NC) I got an internal server error
**To Reproduce**
Updated Collabora app in nextcloud. 

I have 3 nextcloud instances that access to a unique collabora instance. ALL are working fine except the nextcloud that I updated to Collabora 3.3.0 app.

affected nextcloud instance works perfect, but If I open a document that use Collabora I get the error. Pdf, etc works fine.

Collabora admin has disappeared from admin menu, and url /index.php/settings/admin/richdocuments does not open anything

Tried to remove and reinstall collabora, problem persist

**Client details:**
tested in multiple devices, including mobile, linux and windows with different browsers

## Server details

**Operating system**: Ubuntu 16.04 LTS

**Web server:** apacha 2.4

**Database:** Mysql

**PHP version:** 7.0.33

**Nextcloud version:** 15.0.7

**Version of the richdocuments app**

**Version of Collabora Online**

<details>
<summary>Logs</summary>

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
Error	index	OCP\AppFramework\QueryException: Could not resolve OCA\Richdocuments\Controller\DocumentController! Class OCA\Richdocuments\TemplateManager does not exist

    /var/www/testnc/lib/private/AppFramework/Utility/SimpleContainer.php - line 119:

    OC\AppFramework\Utility\SimpleContainer->resolve("OCA\\Richdo ... r")

    /var/www/testnc/lib/private/AppFramework/DependencyInjection/DIContainer.php - line 426:

    OC\AppFramework\Utility\SimpleContainer->query("OCA\\Richdo ... r")

    /var/www/testnc/lib/private/AppFramework/DependencyInjection/DIContainer.php - line 397:

    OC\AppFramework\DependencyInjection\DIContainer->queryNoFallback("OCA\\Richdo ... r")

    /var/www/testnc/lib/private/AppFramework/App.php - line 105:

    OC\AppFramework\DependencyInjection\DIContainer->query("OCA\\Richdo ... r")

    /var/www/testnc/lib/private/AppFramework/Routing/RouteActionHandler.php - line 47:

    OC\AppFramework\App::main("OCA\\Richdo ... r", "index", OC\AppFramew ... {}, { _route: "r ... "})

    <<closure>>

    OC\AppFramework\Routing\RouteActionHandler->__invoke({ _route: "r ... "})

    /var/www/testnc/lib/private/Route/Router.php - line 297:

    call_user_func(OC\AppFramew ... {}, { _route: "r ... "})

    /var/www/testnc/lib/base.php - line 987:

    OC\Route\Router->match("/apps/richdocuments/index")

    /var/www/testnc/index.php - line 42:

    OC::handleRequest()
#### Browser log
```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log
c) ...
```

</details>

**Comment by supremesyntax** [comment]
Same here

**Comment by juliushaertl** [comment]
A fix for this has just been merged in #488

**Comment by ralcazar-oeg** [comment]
Update to 3.3.1 solved the issue. Thank you!
